billsplit/src/components/Invoice.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+
+
+const Invoice = (props) => {
+    return (
+        <>
+            <div className="ui raised segment container">
+                <div className="ui large header">
+                    For the event of {props.eventTitle}
+                </div>
+                <div className="ui header">
+                    Your total is ${props.amountTotal}
+                </div>
+                <div className="ui header">
+                    With a party of {props.numberOfPartyMembers}
+                </div>
+                <div className="ui header">
+                    {props.taxPercent ? `Tax {props.taxPercent}%` : null}
+                </div>
+                <div className="ui header">
+                    {props.tipPercent ? `Tip {props.tipPercent}%`: null}
```

Comment:
Same issue here. The proper syntax is 
```
`Tip ${props.tipPercent}%`
```

Reply:
yeah i just fixed this a few mintues, great catch